roles/nodepool/files/etc/nodepool/elements/nodepool/cleanup.d/90-validate-dib-image

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+function reporter () {
+  message="$1"
+  shift
+  echo
+  echo "${message}"
+  for (( i=0; i<${message}; i++ )); do
+    echo -n '-'
+  done
+  echo
+}
+
+function build_failure () {
+  reporter "Error encountered! Exiting build."
+  reporter "${message}" >&2
+  exit 1
+}
+
+function check_python () {
+  reporter "Validating python installation."
+  python_version=$(python --version)
+  if [[ ! "$python_version" == *"Python 2.7" ]]; then
+    build_failure "Python does not exist or is not version 2.7.x"
+  fi
```

Comment:
@gandelman-a I wasn't sure how this would play out in the future, so just went with what we need now. I can easily just change it to a "do we have any python?" check.